src/main/java/com/lenis0012/bukkit/marriage2/commands/CommandGift.java

```diff
@@ -31,14 +31,14 @@ public void execute() {
             return;
         }
 
-        ItemStack item = player.getItemInHand();
+        ItemStack item = player.getInventory().getItemInMainHand();
```

Comment:
Pretty sure that this does not work on 1.8. I would like to get at least 1 more stable 2.0.x build out before ditching 1.8 support and working on new features.

Reply:
Yeah, half of the intent here was to ditch 1.8. I can see your reservations with that, though.